integrations_example.py

```diff
@@ -8,43 +8,55 @@
 # Version:      1.0
 #
 
-import json
 import argparse
 import csv
 import demisto
 
+
 def p(what):
     if verbose:
         print(what)
 
+
 def options_handler():
     parser = argparse.ArgumentParser(description='Integrations and commands')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
-    parser.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True, help="no extra prints")
-    parser.add_argument('-o', '--output', help='The output CSV file', default='integrations.csv')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
+    parser.add_argument(
+        '-q', '--quiet', action='store_false',
+        dest='verbose', default=True, help="no extra prints")
```

Comment:
`default` is already set to `True`...

Reply:
removed default option from where store_true of store_false is set.(e44c2f8)